include/xtensor/xbuilder.hpp

```diff
@@ -135,7 +135,7 @@ namespace xt
             }
         };
 
-        template <class T, template<class, class> class F>
+        template <class T, template <class> class K, class F = K<T>>
```

Comment:
I changed things a bit more to accommodate for the `k` parameter. Let me know what you think.